app/move/views/cancel.njk

```diff
@@ -5,6 +5,9 @@
 {% endblock %}
 
 {% block pageTitle %}
```

Comment:
Only thing I was wondering is whether it was worth creating a separate block for the page title for views that inherit from the form wizard to override the page title but to keep the error prefix always in place and it one place.

It might avoid the same scenario accidentally happening again if we override it in another sub-template. Or do you think that's overkill?

Reply:
good idea - done 👍